bin/reth/src/metrics_describer.rs

```diff
@@ -0,0 +1,24 @@
+//! Metrics describer
+
+use metrics::{describe_counter, describe_histogram};
+
+pub(crate) fn describe_metrics() {
+    // Describe stagedsync headers metrics
+    describe_counter!("stages.headers.counter", "Number of headers successfully retrived");
```

Comment:
hmm, not entirely sure if this should be here. we should find a way to describe the metrics in the crates they belong to for maximum reusability